corehq/apps/export/tests/test_export_views.py

```diff
@@ -118,7 +134,7 @@ def test_data_file_download_expired(self):
     (1000, 1999),
     (11000, None)  # This test uses this file (test_export_views.py). `11000` must be less than its size.
 ], DataFileDownloadDetailTest)
-@flag_enabled('DATA_FILE_DOWNLOAD')
+@domain_has_dropzone()
```

Comment:
I think you can use `corehq.util.test_utils.privilege_enabled` decorator for this instead of the one-off domain_has_dropzone

Reply:
It didn't work for me because it patches `django_prbac.decorators.has_privilege` instead of `corehq.apps.users.permissions.domain_has_privilege`.

Reply:
Maybe I should make `corehq.util.test_utils.privilege_enabled` patch `domain_has_privilege()` in the places I need. (Two more places that it's currently patching.) I do like the idea of reusing existing utils. What do you think?

Reply:
1760afc8fdd6af5b886f37d24311bc7bb48185e1